src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -256,17 +256,19 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const backButtonOptionList = ownedVenues.filter(
-    ({ id, name, template, worldId }) => {
-      if (venueId === id || worldId !== roomVenue?.worldId) {
-        return null;
-      }
-
-      return {
-        name,
-        template,
-      };
-    }
+  const backButtonOptionList = useMemo(
+    () =>
+      ownedVenues.filter(({ id, name, template, worldId }) => {
```

Comment:
if `ownedVenues.` is `undefined` this will throw an error, not going to get to `??`. 

It can be solved by `ownedVenues?.filter() ?? ALWAYS_EMPTY_ARRAY` or it can be `Array.isArray(ownedVenues) ? ownedVenues.filter() : ALWAYS_EMPTY_ARRAY` (or similar)

Reply:
added a `?` mark in `ownedVenues?.filter`, so should be all good now 4c0fee4